legal-api/report-templates/continuationApplication.html

```diff
@@ -24,29 +24,28 @@
          </table>
          <div class="business-name-header">
             <label class="lbl-business-name">
-            {% if nameRequest.legalName %}
-               {{ nameRequest.legalName }}
-            {% else %}
-               {{ numberedDescription }}
-            {% endif %}
+               {% if business.legalName %}
+                  {{ business.legalName }}
+               {% else %}
+                  {{numberedDescription}}
+          {% endif %}
             </label>
          </div>
       </div>
       [[common/businessDetails.html]]
 
       <div class="section-header mt-5 pt-4">CONTINUATION APPLICATION</div>
-      <div class="container pt-4">
-         [[incorporation-application/nameRequest.html]]
+      <div class="container pt-2">
+         [[continuation/nameRequest.html]]
          [[continuation/effectiveDate.html]]
          [[continuation/foreignJurisdiction.html]]
          [[continuation/exproRegistrationInBc.html]]
          [[continuation/authorization.html]]
-         [[incorporation-application/completingParty.html]]
       </div>
       <div class="section-header mt-7 pt-4">NOTICE OF ARTICLES</div>
-      <div class="container pt-4">
+      <div class="container pt-2">
          [[common/statement.html]]
-         [[incorporation-application/benefitCompanyStmt.html]]
+         [[continuation/companyNameStmt.html]]
```

Comment:
since you revert the changes, we don't need a new file you can refer back to `[[incorporation-application/benefitCompanyStmt.html]]`

Reply:
incorporation-application/benefitCompanyStmt.html will show 'A company name using the Incorporation Number will be provided..' for all numbered companies, even those already got the legal name.
![image](https://github.com/user-attachments/assets/63299c6a-50e6-4651-9377-f3d0b146cb1d)
it didn't check the business.legal name
Maybe we update incorporation-application/benefitCompanyStmt.html ?

Reply:
Deleted the new file, refer back to [[incorporation-application/benefitCompanyStmt.html]]
And I updated [[incorporation-application/benefitCompanyStmt.html]] to check business.legal name